soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -180,6 +181,11 @@ void power_gate_exit(void)
 	cpu_early_init();
 	sys_cache_data_flush_and_invd_all();
 	_restore_core_context();
+
+	/* Secondary core is resumed by set_dx */
+	if (arch_proc_id()) {
+		mp_resume_entry();
+	}
```

Comment:
This has been causing regressions and test failures (https://github.com/thesofproject/sof/issues/8818). Tentative removal of this part in:
- #68366